test/Microsoft.Net.Http.Headers.Tests/CookieHeaderValueTest.cs

```diff
@@ -154,7 +154,7 @@ public void CookieHeaderValue_Ctor2InitializesCorrectly(string name, string valu
         public void CookieHeaderValue_Value()
         {
             var cookie = new CookieHeaderValue("name");
-            Assert.Equal(String.Empty, cookie.Value);
+            Assert.Equal(string.Empty, cookie.Value);
```

Comment:
We use it elsewhere in our tests. I'd leave it up to you since it doesn't change the intent.